docs/en/toolkit/toolkit_basics_guides/advanced_config.md

```diff
@@ -7,7 +7,7 @@ lang: en
 
 # Getting started with configurations
 
-After completing this guide, you will have the knowledge fundamental to extending and customizing a configuration for your project’s pipeline.
+After completing this guide, you will have the knowledge fundamental to adding your project information to a Default Configuration, associating that configuration with your project, and preparing your pipeline configuration to be customized.
 
```

Comment:
Again, we're not creating a Default Config here, just creating a config for our project based on the Default Config. How about this: 

After completing this guide, you will have the knowledge fundamental to creating an editable configuration for your project based on the Default Configuration.

Reply:
I'm a bit confused. I thought the Wizard allows the user to add their project information to a default config etc... exactly what I wrote. I thought that sounded good and is exactly what is being done. There's a Default configuration that the wizard is editing to add the users project information to. I don't say you're creating a default configuration anywhere.